source/modules.tex

```diff
@@ -201,28 +202,33 @@
 or indirectly within an unnamed namespace
 or a \grammarterm{private-module-fragment}.
 An \grammarterm{export-declaration}
-has the declarative effects of its \grammarterm{declaration}
-or its \grammarterm{declaration-seq} (if any).
+has the declarative effects of its
+\grammarterm{declaration},
+\grammarterm{declaration-seq} (if any), or
+\grammarterm{module-import-declarartion}.
 An \grammarterm{export-declaration} does not
 establish a scope and its \grammarterm{declaration}
 or \grammarterm{declaration-seq}
-shall not contain an \grammarterm{export-declaration}.
+shall not contain an \grammarterm{export-declaration} or
+\grammarterm{module-import-declaration}.
 
 \pnum
 A declaration is \defnx{exported}{declaration!exported} if it is
 \begin{itemize}
 \item a namespace-scope declaration declared within an
       \grammarterm{export-declaration}, or
-\item a \grammarterm{module-import-declaration} declared with
-      the \tcode{export} keyword\iref{module.import}, or
+\item a \grammarterm{module-import-declaration}
+      declared within an \grammarterm{export-declaration}.
```

Comment:
Please follow the wording and delete "declared".  Keeping the "declared" here is a normative change, which as I read it, changes the meaning from something that was purely grammatical to whether an export-declaration declares a module-import-declaration.

Reply:
Fixed (forgot to delete "declared").